public/app/plugins/datasource/loki/responseUtils.ts

```diff
@@ -1,4 +1,6 @@
-import { DataFrame, FieldType, getParser, Labels, LogsParsers } from '@grafana/data';
+import { DataFrame, FieldType, Labels } from '@grafana/data';
+
+import { getParser, LogsParsers } from '../../../features/logs/utils';
```

Comment:
The same as above.

Reply:
hmm.. we should have a chat about it.. i just don't know how much these are useful in general. the only datasource that uses it is Loki, to do the add-query-hints thing... i'd love to see another datasource where this thing is useful (to be able to look at a text-string and investigate it for potentially being logfmt or json).

Reply:
Shouldn't we move this to Loki then. As other data sources are not using it.

Reply:
i am sorry, i formulated my answer incorrectly. the only datasource using it is Loki, but it is also used by not-datasource-code, by files in `/public/app/features/logs/components/`, so moving i to Loki is not possibe currently 😿 . would you be ok with the current situation there, or should we discuss this more? thanks!